doc/porting/shields.rst

```diff
@@ -0,0 +1,83 @@
+.. _shields
+
+Shields
+#######
+
+Shields have been introduced on Zephyr as a new element that can be used
```

Comment:
This won't be a "new element" in a few months, so better to say:

    Shields, also known as "add-on" or "daughter boards", attach to a board
    to extend its features and services for easier and modularized prototyping.
    In Zephyr, the shield feature provides Zephyr-formatted shield
    descriptions for easier compatibility with applications.